daprdocs/content/en/dotnet-sdk-docs/_index.md

```diff
@@ -18,7 +18,15 @@ Dapr offers a variety of packages to help with the development of .NET applicati
 
 - [Dapr CLI]({{< ref install-dapr-cli.md >}}) installed
 - Initialized [Dapr environment]({{< ref install-dapr-selfhost.md >}})
-- [.NET 6](https://dotnet.microsoft.com/download) or [.NET 8+](https://dotnet.microsoft.com/download) installed
+- [.NET 6](https://dotnet.microsoft.com/download), [.NET 8](https://dotnet.microsoft.com/download) or [.NET 9](https://dotnet.microsoft.com/download) installed
+
+{{% alert title="Note" color="primary" %}}
+
+Note that while .NET 6 is generally supported as the minimum .NET requirement across the Dapr .NET SDK packages
+and .NET 7 is the minimally supported version of .NET by Dapr.Workflows in Dapr v1.15, only .NET 8 and .NET 9 wil
```

Comment:
"will"

Reply:
Good catch - fixed that here and in the other PR I copy/pasted this into.